index.html

```diff
@@ -63,6 +63,199 @@ <h6>LET'S CONNECT</h6>
+              sign-ups required.
+            </p>
+            <ul class="project_coding_langs">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="preview_button outlinings">See Project</button>
+          </div>
+        </article>
+        </article>
+      </div>
+    </section>
+
+    <section class="projects" id="about-id">
+      <div class="desktop-project-container">
+        <article class="white-back">
+          <img
+            class="mobile-work-img"
+            src="./images/Snapshoot Portfolio4.png"
+            alt="tonic project preview"
+          />
+          <div class="desktop-work-detail">
+            <h3>Multi-Post Stories</h3>
+            <ul class="project-details">
+              <li>CANOPY</li>
+              <li>BACKEND DEV</li>
+              <li>2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="project_coding_langs">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="preview_button outlinings">See Project</button>
+          </div>
+        </article>
+        </article>
+      </div>
+    </section>
+
+    <section class="about-white-back">
+      <div class="desktop-about">
+        <h3>
+          About <br />
+          Myself
+        </h3>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="socials-about">
+          <h6>LET'S CONNECT</h6>
+          <ul class="social-icon">
+            <li>
+              <a href="#"><img src="./images/twitter.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/Linkedin.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/medium.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/github.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/victory.png" alt=""></a>
+            </li>
+          </ul>
+        </div>
+        <button class="outlinings">Get my resume</button>
+      </div>
+      <div class="desktop-lang">
+        <ul class="about__stacks">
+          <li class="about_stacks__item" style="border-bottom: 0px">
+            <span>Languages</span>
+            <i class="fa fa-angle-down"></i>
+          </li>
+          <ul class="lang">
+            <li class="js">
+              <img src="images/js.png" alt="" />
+              <h3>JavaScript</h3>
```

Comment:
- [ ] [ OPTIONAL ] Since these are not headings but texts to show the languages you are skilled in, please consider using a `p` element instead of a `hx`. The same applies to HTML and CSS below. This ensures good UX and accessibility.